### PR TITLE
Merge better-grid-lit-next into master for stopgap 0.6.0 release

### PR DESCRIPTION
**Pull request by graynorton**

### CODE REVIEW

File: packages/lit-virtualizer/README.md [hunk 1]
```diff
@@ -85,7 +86,7 @@ Say we are building an index page for a blog, and need a list of link to all blo
 </body>
 ```
 
-...and here's how we can use the lit-virtualizer's `scroll` directive to render the list of post links.
+...and here's how we can use the `scroll` directive to render the list of post links.
 ```js
 import scroll from 'lit-virtualizer'
```
Comment (bramkragten) [inline packages/lit-virtualizer/README.md]: import scroll from '@lit-labs/virtualizer'

File: packages/lit-virtualizer/README.md [hunk 2]
```diff
@@ -167,9 +162,9 @@ Note: Rendering with `scrollToIndex` will cause the scroll view to fix at the gi
 
 ### Events
 
-#### `rangechange` event
+#### `visibilityChanged` event
```
Comment (bramkragten) [inline packages/lit-virtualizer/README.md:165]: Shouldn't this be `rangechanged`?

File: packages/lit-virtualizer/README.md [hunk 3]
```diff
@@ -187,7 +182,7 @@ const handleEvent = (e) => {
 }
 
 const example = (contacts) => html`
-  <section @rangechange=${handleEvent}>
+  <section @rangechanged=${handleEvent}>
```
Comment (bramkragten) [inline packages/lit-virtualizer/README.md:185]: Or should this be `visibilityChanged`?

Comment (graynorton) [reply packages/lit-virtualizer/README.md]: This. Thanks for the catch!

**Pull request merged**